src/runtime/CMakeLists.txt

```diff
@@ -334,6 +334,8 @@ endforeach ()
 
 add_library(Halide_Runtime INTERFACE)
 add_library(Halide::Runtime ALIAS Halide_Runtime)
-target_include_directories(Halide_Runtime INTERFACE $<BUILD_INTERFACE:${Halide_BINARY_DIR}/include>)
+target_include_directories(Halide_Runtime INTERFACE
+  $<BUILD_INTERFACE:${Halide_BINARY_DIR}/include>
+  $<INSTALL_INTERFACE:include/halide>)
```

Comment:
This is wrong. The `INCLUDES DESTINATION` argument in `packaging/CMakeLists.txt` sets it up correctly already.

Reply:
Aha, that is good to know.